source/Calamari.Shared/Integration/Packages/Download/DockerImagePackageDownloader.cs

```diff
@@ -15,7 +15,7 @@ namespace Calamari.Integration.Packages.Download
 {
     public class DockerImagePackageDownloader : IPackageDownloader
     {
-        readonly IScriptEngine scriptEngine;
+        readonly ICombinedScriptEngine scriptEngine;
```

Comment:
Worth renaming the field as well?

Reply:
I'm going the other way, with the 2nd refactor nothing else is a `ScriptEngine`, so renamed `CombinedScriptEngine` to that.